src/test/java/org/sonarsource/sonarlint/ls/mediumtests/ConnectedModeMediumTests.java

```diff
@@ -1077,6 +1081,15 @@ void shouldReopenResolvedLocalIssues() {
         tuple(2, 2, 2, 7, PYTHON_S1481, "sonarlint", "Remove the unused local variable \"plouf\".", DiagnosticSeverity.Warning)));
   }
 
+  @Test
+  void shouldIgnoreRazorFile() {
+    var uriInFolder = folder1BaseDir.resolve("shouldIgnore.razor").toUri().toString();
+    didOpen(uriInFolder, "csharp", "@using System");
+
+    assertLogContains("'OmniSharp' skipped because there is no related files in the current project");
+    awaitUntilAsserted(() -> assertThat(client.getDiagnostics(uriInFolder)).isEmpty());
```

Comment:
IMHO, this should `awaitUntilAsserted` that the log contains an info message with `Found 0 issues` _and only then_ check for the "skipped" log - I believe that the test as it is now would be flaky.